### ISSUE TITLE
Unable to initiate an upload - no reason given - HTTP status 201 instead 200

### ISSUE THREAD

**Issue by netbrothers-tr** [vimeo/vimeo.php#196]
When we transferred our app from development to production, we've experienced a problem we can't make any sense of. The Vimeo SDK is the same, the app is the same, even the video file is the same. What's different, though, is the user account. The dev account and the production account are both Plus accounts. They both have upload access. Both tokens include all scopes. So there isn't any difference (we practically cloned the users' settings) and there shouldn't be any access issues.

We found differences in the server response, setting a `var_dump($attempt)` right after [this line](https://github.com/vimeo/vimeo.php/blob/master/src/Vimeo/Vimeo.php#L332).

On the development system we immediately receive a 200 HTTP response. On the production system, we receive a 201 HTTP response. Also, on the development system in the response the approach is set to `tus` whereas the approach in the response gotten by the production system is `post`. So we have

Property | Dev  | Production
------------ | ------------- | -------------
Content of `['status']` | 200 | 201
Content of `['body']['upload']['form']` | `NULL` | form HTML
Content of `['body']['upload']['approach']` | `tus` | `post`
Outcome | upload completes | upload is incomplete

To be clear: we did not define the approach. We went with the defaults here. So according to [this line](https://github.com/vimeo/vimeo.php/blob/master/src/Vimeo/Vimeo.php#L326) the client should always try `tus`.

I might add:

1. Videos objects are created with either system but the upload is completed immediately on the development system and stays incomplete on the production system.
2. If we use the form provided in the 201 server response on the production system, we can **manually** complete the upload, providing the video file again in a browser.

What would be the most promising direction to investigate??? We aim for a fully automated process with no user interaction. Thanks for any feedback!

**Comment by netbrothers-tr** [comment]
**TL;DR:** It was an encoding issue in the title and the description of the video.

Oddly the client received a comprehensible error message only when the title and the description fields have been filled with a `boolean` value of `false`, as a result of a failing `json_encode()` call. Without JSON-encoding the call provoked the above behaviour not providing any clue about why the upload was failing.
